runtime/rasdump/trigger.c

```diff
@@ -603,15 +603,15 @@ prepareForDump(struct J9JavaVM *vm, struct J9RASdumpAgent *agent, struct J9RASdu
 		 */
 
 		/* Share exclusive access when it's a "slow entry" or "user" event, as there may be a deadlock */
-		UDATA shareVMAccess = exclusiveHeld &&
-				( (context->eventFlags & J9RAS_DUMP_ON_USER_SIGNAL) ||
-				(context->eventFlags & J9RAS_DUMP_ON_SLOW_EXCLUSIVE_ENTER) );
+		UDATA shareVMAccess = exclusiveHeld
+						&& OMR_ARE_ANY_BITS_SET(context->eventFlags, J9RAS_DUMP_ON_USER_SIGNAL | J9RAS_DUMP_ON_SLOW_EXCLUSIVE_ENTER);
```

Comment:
Please decrease the indentation here: two tabs more than the line above are sufficient.